operatorcert/utils.py

```diff
@@ -0,0 +1,164 @@
+import json
```

Comment:
Should all the functions now be in the `utils`? This sounds like an antipattern to me

Reply:
You're right. I moved most of the functions into the module's `__init__.py`